src/native-common/TextInput.tsx

```diff
@@ -117,7 +132,7 @@ export class TextInput extends React.Component<Types.TextInputProps, TextInputSt
         }
     }
 
-    private _onSelectionChange = (selEvent: React.SyntheticEvent<TextInput>) => {
+    private _onSelectionChange = (selEvent: React.SyntheticEvent<any>) => {
```

Comment:
>any [](start = 65, length = 3)

Why is this being changed?

Reply:
For consistency with other places, and because it hurt my eyes... This is a handler called by an RN.TextInput, so the correct type would have to be React.SyntheticEvent<RN.TextInput>, I think.

---
In reply to: [159349506](https://github.com/Microsoft/reactxp/pull/404#discussion_r159349506) [](ancestors = 159349506)